runner.go

```diff
@@ -6,6 +6,10 @@ import (
 
 var _ = registerGob(&passThrough{}, &pick{})
 
+// SizeUnknown is a special value that should be returned by ApproxSizers when
+// they cannot give an estimation.
```

Comment:
this might be too specific.. for example, we can also use this const in type.Size methods.

what do you think about:
```suggestion
// SizeUnknown is used when size can't be estimated
```